integration/airflow/openlineage/airflow/utils.py

```diff
@@ -231,15 +230,14 @@ def import_from_string(path: str):
         module = importlib.import_module(module_path)
         return getattr(module, target)
     except Exception as e:
-        logging.info(e)
```

Comment:
I don't think we need this, since we report the error again below. But maybe there is a reason for it I am not aware of? This does give a more detailed error.